charts/app-config/image-tags/integration/govspeak-preview

```diff
@@ -0,0 +1,3 @@
+image_tag: latest
+automatic_deploys_enabled: true
+promote_deployment: false
```

Comment:
**question:** promoting deployments

We don't want to promote deployment?

Reply:
This file gets updated by the `govuk-ci` user when we need a deployment to be promoted (along with the correct `image_tag`).  At the moment, it doesn't matter as I'll be manually deploying the first build anyway.